src/report_workflow/README.md

```diff
@@ -0,0 +1,24 @@
+#### Generate test-report manifest for each test.
+The input requires a mandatory `test-manifest-path`, `--artifact-paths`, `--base-path`, `--test-type` and `--test-run-id`, optional `--output-path` and `--component` to automatically generate the test-report manifest after testing.
+
+*Usage*
+```
+./report.sh <test-manifest-path> --artifact-paths opensearch=<...> opensearch-dashboards=<...> --test-run-id <...> --test-type integ-test --base-path <...>
+```
+e.g.
+```
+./report.sh manifests/2.9.0/opensearch-2.9.0-test.yml -p opensearch=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.9.0/8172/linux/x64/tar --test-run-id 5328 --test-type integ-test --base-path https://ci.opensearch.org/ci/dbc/integ-test/2.9.0/8172/linux/x64/tar
+```
+The following options are available.
+
+| name                 | description                                                                    |
+|----------------------|--------------------------------------------------------------------------------|
+| test-manifest-path   | Specify a test manifest path.                                                  |
+| -p, --artifact-paths | Artifact paths of distributions used for testing.                              |
+| --base-path          | Base paths of testing logs.                                                    |
+| --test-type          | Type of tests report generates on.                                             |
+| --output-path        | Specify the path location for the test-report manifest.                        |
+| --test-run-id        | Specify the unique execution id for the test.                                  |
```

Comment:
This test-run-id should be equal to the the test runs build number right?

Reply:
Yes, this id will be used to retrieve the component yml.

Reply:
Can you specify that it should match the test-run-id from already run tests? Seems like you can enter anything which is not right

Reply:
Thanks. Just updated.